apps/beeswax/src/beeswax/conf.py

```diff
@@ -358,3 +358,10 @@ def has_multiple_sessions():
 
 def has_session_pool():
   return has_multiple_sessions() and not CLOSE_SESSIONS.get()
+
+MAX_TABLES_IN_LEFT_ASSIST = Config(
```

Comment:
It's also for DBs and I guess columns, and it's throughout the UI, autocomplete, table browser etc. Perhaps MAX_UI_SQL_ENTRIES is more suitable?

It'd also be good to add something on the lines of "Setting this higher than the default can degrade performance."

Reply:
okay, got it.